### ISSUE TITLE
Laravel 5.5 Tests RefreshDatabase -> This database driver does not support dropping all tables

### ISSUE THREAD

**Issue by ghost** [jenssegers/laravel-mongodb#1318]
Like the title says.

With Laravel 5.4 i ran my tests with 
```
use DatabaseMigrations;
```
without problems.

Now with Laravel 5.5 no matter if i use
```
use DatabaseMigrations;
```
or 
```
use RefreshDatabase
```

i get an exception when i run my tests:
```
LogicException: This database driver does not support dropping all tables.
```

**Comment by vgladimir** [comment]
When this fill be fixed?

**Comment by ebisbe** [comment]
@vgladimir this is fixed but it's in the release v3.4.* so that means that is not included for Laravel 5.5 and only in 5.6... Is this right @jenssegers ??

**Comment by ebisbe** [comment]
@vgladimir I've forked the project and updated the composer to let use Laravel 5.5. If you want to use it, change:
```
"jenssegers/mongodb": "dev-master",
```
Add under "repositories":
```
"repositories": [
        {
            "type": "vcs",
            "url":  "git@github.com:adinton/laravel-mongodb"
        }
]
```

And then **composer update jenssegers/mongodb** . This will use our repository. If you update to 5.6. Remove this changes and use v3.4.*
